gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_4_0/00_add_new_integration_table.yml

```diff
@@ -1,11 +1,8 @@
 databaseChangeLog:
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.